ui/apps/platform/src/Components/PatternFly/FormLabelGroup.tsx

```diff
@@ -19,13 +19,14 @@ function FormLabelGroup<T>({
 }: FormLabelGroupProps<T>): ReactElement {
     const error = get(errors, fieldId);
     const isTouched = touched && get(touched, fieldId);
+    const showError = touched === undefined ? error : isTouched && error;
```

Comment:
This is something worth discussing. Should we show errors only after a form field is touched or should we show the errors instantly? For this feature, I felt like seeing the disabled buttons but not knowing why it's disabled was a bit confusing. The little red dots that signify that a field is required do help, but I felt like just seeing the error immediately helped me spot where to make changes. I also felt like it was a bit of a hassle trying to work with some of the PatternFly components to make it work using the "touched" approach. Considering those points, I felt like it was a better choice to show the errors regardless of whether the form field was touched. Others might not agree with that approach, so wanted to get input

cc: @vjwilson @dvail @pedrottimark @bradr5 @alwayshooin

Reply:
I agree that having the Save button disabled can be confusing for someone if they do not notice the "required" markers.

However, I don't think coming to a form fresh and seeing errors is a good user experience.

A long-term solution would be to have the button enabled, but if you click it and there are errors/things missing, then the app would guide you to the first field that needs attention. I think we should discuss a plan for something like that with the UX team.